packages/forms/src/UIForm/fields/Date/Date.component.js

```diff
@@ -1,114 +1,84 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import memoizeOne from 'memoize-one';
-import InputDateTimePicker from '@talend/react-components/lib/DateTimePickers';
+import { InputDatePicker } from '@talend/react-components/lib/DateTimePickers';
 import FieldTemplate from '../FieldTemplate';
-import { isoDateTimeRegExp } from '../../customFormats';
+import { convertDate, isoStrToDate } from './Date.utils';
 import { generateDescriptionId, generateErrorId } from '../../Message/generateId';
 
-const INVALID_DATE = new Date('');
+const memorizedIsoStrToDate = memoizeOne(isoStrToDate);
 
-function isoStrToDate(isoStr) {
-	if (isoDateTimeRegExp.test(isoStr)) {
-		return new Date(isoStr);
-	}
-	return INVALID_DATE;
-}
-
-function dateToIsoStr(date) {
-	return date.toISOString();
-}
```

Comment:
move these 2 utility functions to `Date.utils.js`, because it will be reused by DateTime widget